CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+### Linee guida per lo sviluppo
+
+ - Board per gestire i task (TODO)
+ - I task devono essere semplici e descrittivi in modo che sia chiaro il lavoro da svolgere
+ - **Non** eseguire il push di commit direttamente verso il branch main
+ - Implementazioni monotematiche: creare un feature branch con nome possibilmente descrittivo (es: id dell’issue + brevissima descrizione, 123-feature-foo)
```

Comment:
concordo con il feature branch, però proporrei un naming differente. Esempio:

```suggestion
 - Implementazioni monotematiche: creare un feature branch con nome possibilmente descrittivo (es: feature/`id dell’issue` + `brevissima descrizione`, e.g. `feature/123-foo`)
```

Il motivo è semplicemente che utilizzando GUI git clients (es. Fork), i vari branch vengono raggruppati automaticamente in base allo scopo. Quindi ci sarà un gruppo feature, integration, hotfix, etc... e anche a primo impatto visivo sono tutti simili. Addirittura volendo tenere la dicitura più asciutta, si potrebbe evitare anche l'id dell'issue e inserirla come commento alla PR, cosi sfruttiamo anche l'abilità di GitHub ad autolinkare le issue alle PR

Reply:
sì per me va bene, sulle nomenclature possiamo decidere quello che preferiamo